modules/ROOT/pages/_partials/include-insights-charts.adoc

```diff
@@ -0,0 +1,386 @@
+Virtual core fraction and count for this Mule app, for example, *0.1 vCores x 1* (= 10% of 1 vCore)
+
+Last Updated::
+Date-time stamp for the last redeployment of this Mule app
++
+This stamp comes from Runtime Manager data.
+//end::app-detail-header[]
+//
+//
+// APP DETAIL: Overview tab
+//tag::app-detail-overview[]
+
+//  shared metric in Overview tab
+include::partial$include-insights-charts.adoc[tag="average-latency"]
+include::partial$include-insights-charts.adoc[tag="p95-latency"]
+include::partial$include-insights-charts.adoc[tag="total-requests"]
+
+//TODO: Messages -> Message (SATHISH)
+Mule Messages Volume::
+Total Mule message count for this app
++
+For information about Mule messages, see xref:mule-runtime::about-mule-event.adoc[].
+
+//  shared metric in Overview tab
+include::partial$include-insights-charts.adoc[tag="request-latency"]
+
+Inbound Request Volume::
+Inbound request count at specific time points
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+//  shared metric in Overview tab
+Failed Inbound Request Volume::
+Count of unsuccessful inbound requests at specific time points
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+//  shared metric in Overview tab
+Mule Messages::
+Mule message count for this Mule app at specific time points
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
++
+For information about Mule messages, see xref:mule-runtime::about-mule-event.adoc[].
+//end::app-detail-overview[]
+//---------------------------------------------------------------------------------//
+//
+//---------------------------------------------------------------------------------//
+// APP DETAIL REQUESTS: Requests tab
+//tag::app-detail-requests[]
+Inbound Request Volume::
+Inbound request count total, which you can group by response time or protocol 
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Outbound Request Volume::
+Outbound request count total, which you can group by response time, protocol, or by the 20 hosts (such as an Object Store or a `cloudhub.io` address) with the most outbound requests 
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+//TODO: requests -> Requests (SATHISH)
+Top Slowest Requests by Path::
+Up to 20 endpoints with the slowest requests
++
+The requests must take more than 25ms. If no data is available, then no requests take more than 25ms to process.
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Request Latency by Endpoint::
+Inbound or outbound latency, in milliseconds, by endpoint
++
+The chart captures the average latency and the latency value of requests at the p50, p75, p90, p95, and p99 percentiles.
+ 
+//end::app-detail-requests[]
+//-----------------------//
+//
+//-----------------------//
+//  APP DETAIL ERRORS: Errors tab
+//tag::app-detail-errors[]
+Failed Outbound Request Volume::
+Number of failed outbound requests at specific time points
```

Comment:
Like before, how about we update this to
`Total number of unsuccessful outbound requests made by the application for the selected time duration`?